plugin/vcs/gitlab/sql_review.go

```diff
@@ -0,0 +1,53 @@
+package gitlab
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	// sqlReviewCIVersion is the version for GitLab SQL review CI.
+	sqlReviewCIVersion = "0.0.3"
+	// CIFilePath is the local path for GitLab ci file.
+	CIFilePath = ".gitlab-ci.yml"
+)
+
+// sqlReviewCIFilePath is the remote file path for GitLab SQL review CI.
+// We can include the remote file path in the GitLab CI.
+// So we can store the script in our own GitHub repo: https://github.com/bytebase/sql-review-gitlab-ci
+var sqlReviewCIFilePath = fmt.Sprintf(
+	"https://raw.githubusercontent.com/bytebase/sql-review-gitlab-ci/%s/sql-review.yml",
```

Comment:
We may want to embed the files or configurations into Bytebase binary because not all GitLab self-hosted deployments can access the public Internet, especially there are restricted firewall around.

Reply:
I see. Updated now